src/NServiceBus.Core/Extensibility/ContextBag.cs

```diff
@@ -10,9 +10,9 @@ public class ContextBag : ReadOnlyContextBag
         /// <summary>
         /// Initialized a new instance of <see cref="ContextBag" />.
         /// </summary>
-        public ContextBag(ContextBag parentBag = null)
+        public ContextBag(ContextBag parentContext = null)
```

Comment:
If we're changing the name of the parameter/field, perhaps `parent` would be a better name, since Bag/Context is a bit redundant?